inception-project/inception
# #4068 - Ability to change interface language (i18n)

**What's in the PR**
* Draft of translation support

**How to test manually**
* Configure the application settings.properties file on french language (locale=fr) and verify that texts are in french

**Automatic testing**

**Documentation**

**Todo**
* [x] Move the strings extracted from the telemetry configuration page back into the HTML files and translate the entire HTML files - to avoid having to disable HTML escaping on the resource models
